images/jenkins-inbound-agent/configs/latest.melange.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: inbound-agent
+  version: 3077.vd69cf116da_6f-3
+  epoch: 0
+  description: a Jenkins agent which can connect to Jenkins using TCP or Websocket protocols
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+      - "*"
+      attestation: TODO
+      license: MIT
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - openjdk-17-jre
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    repositories:
+      - https://packages.wolfi.dev/os
+    packages:
+      - ca-certificates-bundle
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/jenkinsci/docker-inbound-agent/archive/refs/tags/${{package.version}}.tar.gz
+      expected-sha256: 9b24f3eb7e6618e92ef1ba410a29f2590766a264e8c8adff8f2163a267086af1
+  - uses: fetch
+    with:
+      # version of agent and the jar file is different so hardcode for now
+      uri: https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/3077.vd69cf116da_6f/remoting-3077.vd69cf116da_6f.jar
```

Comment:
When you PR the melange config to wolfi it is probably better to split the agent and remoting into separate melange configs, that way we can parameterise both versions.  Currently there's no way to add arbitury parameters to a melange config to parameterise multiple version numbers.  Also using the remoting source (https://github.com/jenkinsci/remoting), rather than fetching the pre built binary?  That way we can generate good quality SBOMs and keep it up to date with wolfi automation.

```
https://github.com/jenkinsci/docker-inbound-agent/archive/refs/tags/${{package.version}}.tar.gz
```